### PR TITLE
Gene Page Changes Made 5/24/2018

### PR DESCRIPTION
**Pull request by johnllopez616**
Well. This took longer than anticipated.

Regarding #620, this update changes the font of the page and removes the superscripted links protruding into the headers of the accordion cards.

Regarding #621, the `jaspar-relay-controller` code is now happily in the `server` file with one caveat: The gene page had to be rewritten using Jade to support the dynamically changing JASPAR link.

Changes were made to `graph.js` to support this change, alongside `/server/app.js`, and `web-client/controllers/main.js.` @mihirsamdarshi Please check to see if any of these changes conflict with your branch, the next time you desire to pull from beta.

I understand that this PR should have consisted of multiple commits, but I wasn't aware of the scope of the changes required.

Additionally, I deleted `web-client/public/js/api-test.html`, a file leftover from the BioDB project not needed.

### CODE REVIEW

**Comment by johnllopez616** [comment]
Pull Request Checklist
- [ ] Travis C.I. build passes
- [ ] All four Demos look as expected when loaded
- [ ] Reload works as expected
- [ ] Loading file with Error brings up error modal, and it looks as expected
- [ ] Loading file with Warning brings up warnings modal
- [ ] Import works for both SIF and GraphML
- [ ] Graph can be exported to SIF and GraphML
- [ ] Print works as expected
- [ ] Restrict graph to viewport works as expected (check/uncheck)
- [ ] Viewport Size Changing works as expected (small/medium/large/fit)
- [ ] Toggle between Grid Layout and Force Graph Layout works as expected
- [ ] Force Graph Parameter Sliders change the number above them and have an effect on the graph
- [ ] Locking/Unlocking/Resetting/Undo Resetting the force graph parameters works as expected
- [ ] Enabling and disabling node coloring works as expected
- [ ] Node coloring options work as expected (selection top/bottom dataset, averaging values, changing max value)
- [ ] Weighted graph loaded with the "default to black edges" option checked appears as expected
- [ ] Hide/Show Weights works as expected (always/never/upon mouseover)
- [ ] Edge Weight Normalization Factor can be changed (set/reset)
- [ ] Gray Edge Threshold can be changed, slider changes the number and has an effect on the graph
- [ ] Checking Show Gray Edges as Dashed works as expected (check/uncheck)
- [ ] D-pad left/right/top/bottom/center works as expected
- [ ] Zoom slider changes number and has effect on graph (viewport and menu)

File: web-client/public/gene/api.js [hunk 1]
```diff
@@ -186,6 +187,7 @@
             };
 
             var parseNCBI = function (data) {
+                console.log(data);
```
Comment (dondi) [inline web-client/public/gene/api.js]: Looks like a leftover debug log.

File: web-client/public/gene/api.js [hunk 2]
```diff
@@ -231,6 +233,7 @@
             };
 
             var parseEnsembl = function (data) {
+                console.log(data);
```
Comment (dondi) [inline web-client/public/gene/api.js]: This too.

File: web-client/public/gene/info.html [hunk 3]
```diff
@@ -252,13 +252,13 @@ <h5 class="mb-0">
...
+                      <dd class="molecularFunction col-sm-9 sgdSource"> </dd>
                       <dt class="col-sm-3">Biological Process: </dt>
-                      <dd class="biologicalProcess col-sm-9"></dd>
+                      <dd class="biologicalProcess col-sm-9 sgdSource"></dd>
                       <dt class="col-sm-3">Cellular Component: </dt>
-                      <dd class="cellularComponent col-sm-9"></dd>
+                      <dd class="cellularComponent col-sm-9 sgdSource"></dd>
                     </dl>
                   </div>
                 </div>
```
Comment (dondi) [inline web-client/public/gene/info.html]: Once the transition is fully tested, this file can go away, yes?

File: web-client/views/info.jade [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+                      dl.row
+                        a#ncbiSource.col-6.col-md-4.ncbi-link.database-link
+                      dl.row
+                        a#jasparSource.col-6.col-md-4.jaspar-link.database-link
+        iframe#iframeFooter(src='http://dondi.github.io/GRNsight/onlyfooter.html', scrolling='no')
+        script(src='https://code.jquery.com/jquery-3.2.1.min.js', integrity='sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=', crossorigin='anonymous')
+        script(src='https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js', integrity='sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4', crossorigin='anonymous')
+        script(src='https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js', integrity='sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1', crossorigin='anonymous')
+        script(src='gene/api.js')
+        script(src='gene/info.js')
```
Comment (dondi) [inline web-client/views/info.jade]: Nicely done. More of a "transcription" rather than a "rewrite," I think. Still, a good chunk of work.

**Comment by johnllopez616** [comment]
Any reason coveralls is taking this long to finish?

File: web-client/public/gene/info.html (hunk 3 above)
Comment (johnllopez616) [reply web-client/public/gene/info.html]: Yes. I would feel most comfortable with removing it with the *next* pull request, however, to ensure that the changes work entirely on the `beta` branch.

**Comment by dondi** [comment]
It might just be a reporting glitch; if you click on _Details_ the build looks done: https://coveralls.io/builds/17159043

File: web-client/public/gene/info.html (hunk 3 above)
Comment (dondi) [reply web-client/public/gene/info.html]: Sounds good; keep it until you are comfortable that it's safe to delete. Just wanted to confirm the intent.

**Comment by johnllopez616** [comment]
@dondi or @mihirsamdarshi  Please refresh the server.

**Comment by mihirsamdarshi** [comment]
server refreshed @johnllopez616

**Pull request merged**